### PR TITLE
Update latestRelease.md for 0.24 release

### PR DESCRIPTION
**Pull request by pmhayward**
Signed-off-by: Peter Hayward <pmhayward@uk.ibm.com>

### CODE REVIEW

**Comment by pmhayward** [comment]
Jenkins website stage

File: news-page-content/latestRelease.md [hunk 1]
```diff
@@ -22,26 +22,36 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
 - [OpenJDK version 8](https://adoptopenjdk.net/releases.html?variant=openjdk8&jvmVariant=openj9)
 - [OpenJDK version 11](https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=openj9)
 - [OpenJDK version 15](https://adoptopenjdk.net/releases.html?variant=openjdk15&jvmVariant=openj9)
 
-This release of OpenJ9 sees several performance enhancements: 
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://www.eclipse.org/openj9/docs/openj9_support/).
 
-In this release, we've improved the effectiveness of the option `-XX:[+|-]PortableSharedCache` by also making the AOT compiled code to be more portable across different heap sizes in compressed references mode on the X86 platform.
+This release of OpenJ9 sees several performance enhancements:
```
Comment (SueChaplain) [inline news-page-content/latestRelease.md:38]: Most of these are not performance enhancements, so maybe group them (performance, compatibility, diagnosing problems).

File: news-page-content/latestRelease.md [hunk 2]
```diff
@@ -22,26 +22,36 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://www.eclipse.org/openj9/docs/openj9_support/).
 
-In this release, we've improved the effectiveness of the option `-XX:[+|-]PortableSharedCache` by also making the AOT compiled code to be more portable across different heap sizes in compressed references mode on the X86 platform.
+This release of OpenJ9 sees several performance enhancements: 
+
+
+For compatibility with the reference implementation, we've added support for the `JAVA_OPTIONS` environment variable. You can use this environment variable, which can be overridden by `OPENJ9_JAVA_OPTIONS`, to set command line options.
+
+From OpenJDK 15, the `-XX:[+|-]ShareAnonymousClasses` option enables and disables the storage of hidden classes in the shared classes cache. (In previous versions, it enables and disables the storage of VM anonymous classes in the shared classes cache. )
```
Comment (SueChaplain) [inline news-page-content/latestRelease.md:43]: I found this one confusing, and when I read the docs I found the release topic entry confusing too. I think that "In earlier releases of OpenJ9, this option enables and disables the storage of VM anonymous classes in the shared classes cache. From OpenJ9 0.24.0 for OpenJDK 15 binaries, this option controls the storage of hidden classes too"

File: news-page-content/latestRelease.md (hunk 1 above)
Comment (pmhayward) [reply news-page-content/latestRelease.md]: Was anticipating some performance enhancements, but nothing to highlight; this line removed. All diagnostic changes now grouped together.

File: news-page-content/latestRelease.md (hunk 2 above)
Comment (pmhayward) [reply news-page-content/latestRelease.md]: Updated

**Comment by pmhayward** [comment]
Jenkins website stage

**Pull request merged**